arm/Microsoft.AnalysisServices/servers/readme.md

```diff
@@ -58,8 +71,6 @@ Tag names and tag values can be provided as needed. A tag can be left without a
 
 ### Parameter Usage: `roleAssignments`
 
-Create a role assignment for the given resource. If you want to assign a service principal / managed identity that is created in the same deployment, make sure to also specify the `'principalType'` parameter and set it to 'ServicePrincipal'. This will ensure the role assignment waits for the principal's propagation in Azure.
```

Comment:
This is a bit unfortunate. The change was merged in a different PR, but the update to the ReadMe template is part of [this](https://github.com/Azure/ResourceModules/pull/1185/files#diff-ee3e65b7b8785e56c41f9dac6311e881f1a9104582d7eba8f125d07acc9a60ed) PR.

I'd suggest to either incorporate the change here too (and re-run the generation), or merge the other PR first.

Reply:
Done